app/models/services/base-service.ts

```diff
@@ -318,6 +335,7 @@ export default abstract class BaseService<ServiceParamType> {
               this.config.data_operation_version,
               step.operations || [],
             ),
+            hasAggregatedOutput: _stepHasAggregatedOutput(step),
```

Comment:
I think we're marking this flag on the step that needs to aggregate all the outputs from the prior step's work items. In that case should it be `aggregateInputs`? Well I guess when the step completes it does have aggregated outputs... Names are hard, maybe yours is good.

Reply:
Yeah - I was dithering about that as well, but the ticket says "When services set the aggregated output flag in services.yml, the saved job record indicates its aggregated output status"